## Problem Statement 1:
Blood glucose levels for obese patients have a mean of 100 with a standard deviation of 15. A researcher thinks that a diet high in raw cornstarch will have a positive effect on blood glucose levels. A sample of 36 patients who have tried the raw cornstarch diet have a mean glucose level of 108. Test the hypothesis that the raw cornstarch had an effect or not.

H0: The raw cornstarch had no effect

H1: The raw cornstarch has positive effect

- Upper tail test
- Since n greater than 30 and population varaince is known, we can use Z_test for testing the hypothesis

In [15]:
Z_test = (108 - 100)/(15/6)
Z_test

3.2

P_value = Pr( z > Z_test) = Pr(z > 3.2) = 1 - 0.9993 = 0.0007

Since P_value is lower than 0.05 we can reject the null hypothesis. Thus, the raw cornstarch has positive effect on obese patients

## Problem Statement 2:
In one state, 52% of the voters are Republicans, and 48% are Democrats. In a second
state, 47% of the voters are Republicans, and 53% are Democrats. Suppose a simple
random sample of 100 voters are surveyed from each state.
What is the probability that the survey will show a greater percentage of Republican
voters in the second state than in the first state?

In [8]:
import math
n1 = 100
n2 = 100
R1 = 0.52            # Republicans in  state 1
D1 = 0.48            # Democrats in state 1
R2 = 0.47            # Republicans in state 2
D2 = 0.53            # Democrats in state 2

mu = R1 - R2
print(f"mu: {mu}")

std = math.sqrt(((R1 * D1 ) / n1) + ((R2 * D2) /n2))
print(f"std: {std}")

#This problem requires us to find the probability that p1 is less than p2. 
#This is equivalent to finding the probability that R1 - R2 < 0. 
x = 0

#To find this probability, we need to transform the random variable (R1 - R2) into a z-score. 
#That transformation appears below.

Z_R1_R2 = ( x - mu)/std
print(f"Z_p1_p2 : {Z_R1_R2}")


mu: 0.050000000000000044
std: 0.07061869440877536
Z_p1_p2 : -0.7080278164104213


- From Z table we find that the probability of a z-score being -0.7082 or less is 0.24

- Thus, the probability that the survey will show a greater percentage of Republican voters  in the second state than in the first state is 0.24

## Problem Statement 3:
You take the SAT and score 1100. The mean score for the SAT is 1026 and the standard
deviation is 209. How well did you score on the test compared to the average test taker?

In [1]:
mean = 1026
sigma = 209
x = 1100

z_test = (x - mean)/sigma
z_test

0.35406698564593303

- Z-score of my test score is around 0.35
- Pr(z > z_test) = 0.3631
- So, Person with SAT score 1100 have ranked among the top 37%